templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -151,10 +151,9 @@
 
 	{{if .IsProjectsEnabled}}
 		<div class="divider"></div>
-
-		<div class="ui {{if or (not .HasIssuesOrPullsWritePermission) .Repository.IsArchived}}disabled{{end}} floating jump select-project dropdown">
+		<div class="ui {{if or (not .HasIssuesOrPullsWritePermission) .Repository.IsArchived}}disabled{{end}} floating jump select-projects dropdown">
 			<a class="text muted flex-text-block">
-				<strong>{{ctx.Locale.Tr "repo.issues.new.projects"}}</strong>
+				<strong>{{ctx.locale.Tr "repo.issues.new.projects"}}</strong>
```

Comment:
Why change `ctx.Locale.Tr` to `ctx.locale.Tr`?

Reply:
not sure, this is likely something that was done by @tyroneyeh

Reply:
`ctx.Locale.Tr` is from https://github.com/go-gitea/gitea/pull/27231 changed, `ctx.locale.Tr` is old version's